operator/api/loki/v1/lokistack_types.go

```diff
@@ -1503,4 +1505,4 @@ func (t BlockedQueryTypes) String() string {
 	}
 
 	return strings.Join(res, ",")
-}
```

Comment:
Can you re-add the newline, usually in Linux and Unix-like operating systems, text files are conventionally expected to end with a newline.